cloud_info_provider/providers/openstack.py

```diff
@@ -132,6 +132,8 @@ def get_compute_shares(self, **kwargs):
         shares = self.static.get_compute_shares(prefix=True)
         for share in shares.values():
             share['project'] = share.get('auth', {}).get('project_id')
+            share['default_network_type'] = share.get('default_network_type')
```

Comment:
what about OpenNebula? Shouldn't this be for both?

Reply:
Removed, static provider change is enough.